APIs_Drivers/DigitalInOut_ex_1/README.md

```diff
@@ -0,0 +1,6 @@
+# DigitalInOut example
+
+You can use the DigitalInOut API to both read and write to a digital pin:
+
+1. Use the `output()` and `input()` function calls to switch modes.
+1. Use just like you would DigitalIn or DigitalOut.
```

Comment:
1. and 1. ?

Reply:
Yes, markdown numbers automatically if you put all 1s in the raw file.